tests/integration/tests/admin-areas/admin-areas.test.ts

```diff
@@ -0,0 +1,58 @@
+import * as request from 'supertest';
+
+import { AdminLevel } from '../../helpers/API-service/enum/admin-level.enum';
+import { DisasterType } from '../../helpers/API-service/enum/disaster-type.enum';
+import { FloodsScenario } from '../../helpers/API-service/enum/mock-scenario.enum';
+import { getAccessToken, getServer, mock } from '../../helpers/utility.helper';
+
+export function getAdminAreas(
+  countryCodeISO3: string,
+  disasterType: DisasterType,
+  adminLevel: AdminLevel,
+  accessToken: string,
+): Promise<request.Response> {
+  return getServer()
+    .get(`/admin-areas/${countryCodeISO3}/${disasterType}/${adminLevel}`)
+    .set('Authorization', `Bearer ${accessToken}`);
+}
+
+export default function adminAreaTests() {
+  const countryCodeISO3 = 'UGA';
+  const disasterType = DisasterType.Floods;
+  const adminLevel = AdminLevel.adminLevel3;
+  const scenario = FloodsScenario.Trigger;
+
+  describe('admin areas', () => {
+    let accessToken: string;
+
+    beforeAll(async () => {
+      accessToken = await getAccessToken();
+    });
+
+    it('should return list of admin areas on GET', async () => {
+      // Arrange
+      await mock(scenario, disasterType, countryCodeISO3, null, accessToken);
+
+      // Act
+      const adminAreas = await getAdminAreas(
+        countryCodeISO3,
+        disasterType,
+        adminLevel,
+        accessToken,
+      );
+
+      // Assert
+      expect(adminAreas.status).toBe(200);
+      expect(adminAreas.body.features.length).toBe(23);
```

Comment:
what is good practice about the expected number 23 here? Leave a comment explaining this? Assign a variable expectedNr=23 to at least give a bit more labeling? Work with a full .toMatchSnapshot assertion instead of just this?

Reply:
@jannisvisser I tried `toMatchSnapshot` and found it convenient. But it created a file with 5319 lines for only 2 API calls `/admin-areas` and `/admin-areas/aggregates`. This option will exponentially bloat the repo.

I'll revert back to the hardcoded magic numbers with comments to explain how to interpret them. I'm open to alternatives.

Reply:
yeah that sounds about right. We could think of start using this for shorter responses or just to test specific parts of responses.